### PR TITLE
Scd api update anra

### PR DESCRIPTION
**Pull request by svishnu-flyanra**
Changed docker version to 0.3.17

Changed the API tags in the `scd/test_* to API_0_3_17`

Added scopes from new API in `scd.py`

Changed most of the tests in `test_constrain_simple.py`

Tagged 4 tests in  `test_constrain_simple.py` that fail because of url unsafe endpoints

### CODE REVIEW

File: monitoring/prober/scd/test_constraint_simple.py [hunk 1]
```diff
@@ -239,14 +259,35 @@ def test_mutate_constraint(ids, scd_api, scd_session):
...
+    assert resp.status_code == 200, resp.content
 
-  resp = scd_session.put('/constraint_references/{}'.format(id), json=req, scope=SCOPE_CI)
-  assert resp.status_code == 403, resp.content
+  if scd_api == scd.API_0_3_17:
+    ovn = existing_constraint["ovn"]
 
-  resp = scd_session.put('/constraint_references/{}'.format(id), json=req, scope=SCOPE_CM)
-  assert resp.status_code == 200, resp.content
+    # FIXME: OVNs are not URL safe
```
Comment (BenjaminPelletier) [inline monitoring/prober/scd/test_constraint_simple.py:275]: This should now be resolved with #564

File: monitoring/prober/scd/test_constraint_simple.py [hunk 2]
```diff
@@ -21,11 +21,6 @@
 CONSTRAINT_TYPE = register_resource_type(1, 'Single constraint')
 
 
-def test_set_test_owner_ids(test_owner):
```
Comment (BenjaminPelletier) [inline monitoring/prober/scd/test_constraint_simple.py:24]: We'll need to sync once more from scd_api_update to grab #565 which will remove this block

**Review by BenjaminPelletier — COMMENTED** [review]
Looks good; just have to sync with scd_api_update once more

File: monitoring/prober/scd/test_constraint_simple.py (hunk 1 above)
Comment (svishnu-flyanra) [reply monitoring/prober/scd/test_constraint_simple.py:275]: It is. All the tests pass now!

File: monitoring/prober/scd/test_constraint_simple.py (hunk 2 above)
Comment (svishnu-flyanra) [reply monitoring/prober/scd/test_constraint_simple.py]: synced!

**Review by BenjaminPelletier — APPROVED** [review]

**Pull request merged**